src/AdTypes/ThreshingMachineXml.php

```diff
@@ -43,6 +43,18 @@ public function __set($name, $value)
 
     public function setSegment($segment)
     {
-      // Empty on purpse. There is no segment.
+      // Empty on purpose. There is no segment.
+    }
+
+    public function addEquipment($equipment_items)
+    {
+        // Remove initial empty tag.
+        $this->customTags['TRESHING_MACHINE_EQUIPMENT']->remove();
+        foreach ($equipment_items as $equipment_value) {
```

Comment:
Would be super nice to add a test for this. Would you know how to? Do you want me to help?

Reply:
It's a bit urgent PR, so probably it will be better, if you'll help :sweat_smile: